semgrep-core/src/core/Xlang.mli

```diff
@@ -21,6 +21,9 @@ val of_lang : Lang.t -> t
 (* raises an exception with error message *)
 val to_lang : t -> Lang.t
 
+(* Does not raise, but returns empty list for all but the L variant *)
+val to_langs : t -> Lang.t list
```

Comment:
One idea: `to_lang_exn` for the previous?

Reply:
Good idea. I'll put it in a followup PR since it's mostly unrelated to this one, though.